tests/unit/source_code/samples/simple_notebook.py

```diff
@@ -59,3 +59,7 @@
 # COMMAND ----------
 
 # MAGIC %lsmagic
+
+# COMMAND ----------
+
+# MAGIC %run ./test
```

Comment:
Magic run was missing in example notebook test, combined with the expected output in the yaml below